File: array.c
#include <stdio.h>



/**
 * main - A program that will read an array of 200 characters and display to the screen a count of the occurrences of 
 * each of the five vowels (a, I , e ,o , u) in an array
 * @return : Return 0 on success
 */
int main(){
    char str[200];
    int a_count = 0, e_count = 0, i_count = 0, o_count = 0, u_count = 0;
    
    printf("Enter a string (max 200 characters): ");
    fgets(str, 200, stdin); 
    
    /* Loop */
    for (int j = 0; str[j] != '\0'; j++) {
        char ch = str[j];
        switch (ch) {
            case 'a':
            case 'A':
                a_count++;
                break;
            case 'e':
            case 'E':
                e_count++;
                break;
            case 'i':
            case 'I':
                i_count++;
                break;
            case 'o':
            case 'O':
                o_count++;
                break;
            case 'u':
            case 'U':
                u_count++;
                break;
            default:
                break;
        }
    }
    
    printf("\nVowel counts:\n"
    );
    printf("A/a: %d\n", a_count);
    printf("E/e: %d\n", e_count);
    printf("I/i: %d\n", i_count);
    printf("O/o: %d\n", o_count);
    printf("U/u: %d\n", u_count);

    return 0;

    /*.....END.....*/
}

File: array.py
# Name - Obam Isaiah Olohu
# Dept - Computer Science
#level - 200l
# main - A program that will read an array of 200 characters and
#  display to the screen a count of the occurrences 
# of each of the five vowels (a, I , e ,o , u) in an array.....

# @return :  Return 0 on sucess






# create a function
def count_vowels(input_str):
    # Initialize vowel counters
    a_count = e_count = i_count = o_count = u_count = 0

    # Loop through each character in the string
    for ch in input_str:
        if ch.lower() == 'a':
            a_count += 1
        elif ch.lower() == 'e':
            e_count += 1
        elif ch.lower() == 'i':
            i_count += 1
        elif ch.lower() == 'o':
            o_count += 1
        elif ch.lower() == 'u':65
            u_count += 1

    # Print the counts of vowels
    print(f"A/a: {a_count}")
    print(f"E/e: {e_count}")
    print(f"I/i: {i_count}")
    print(f"O/o: {o_count}")
    print(f"U/u: {u_count}")

# Read a string of up to 200 characters from the user
input_str = input("Enter a string (max 200 characters): ")[:200]

# you cann the function here
count_vowels(input_str)
